Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Add souls as a currency

## Changelog:
- Added Souls as a currency. They are gained by killing people.


## Tests:


# How to test:
* You can view current amount of souls in the debug decisions (wild_goose_chase_decision, add_soul, remove_soul).
* Try to kill someone (Quick imprison -> execute) and see if the number increments
